### DISCUSSION TITLE
Reference for momepy

### DISCUSSION THREAD

**Discussion by cdkang** [pysal/momepy#375] [Q&A] [answered]
Hello, 

Thank you for letting me operate momepy for my research. 

I am wondering if there are books or other comprehensive manuals for arranging the concept and meaning of each momepy functionality. 

For example, the definition and meaning of equivalent rectangular index related to bounding rectangle. I also want to know the concept of  bounding rectangle. 

Understaning momepy functionality requires the knowledge of architecture, urban structure, transportation, design, and other related fields. 

Sincerely,

**Comment by martinfleis** [comment, marked as answer]
Hi, most of the functions used in momepy are based on some academic literature. A complete list is available in [the documentation](http://docs.momepy.org/en/stable/references.html). You can often see the links to related paper in the API of individual functions, like the ERI you mention here http://docs.momepy.org/en/stable/generated/momepy.EquivalentRectangularIndex.html. 

Regarding concepts like minimum bounding rectangle, that is more of a GIS stuff and I suggest simply googling individual concepts if you don't know what they mean. I am not aware of a guide but I never looked for one.

**Comment by cdkang** [comment]
Hello, 

Thank you for your information. 

Sincerely,
